README.md

```diff
@@ -56,6 +56,7 @@ $CONFIG = array (
     //   * home:     Home directory location. A symlink or external storage to this location is used
     //   * ldap_uid: LDAP uid to search for when running in proxy mode
     //   * groups:   Array or space separated string of NC groups for the user
+    //   * photoURL: The URL of the user avatar
```

Comment:
We should likely state here explicitly that the Nextcloud server will make a request to the specified URL to fetch the image, which can potentially expose internal services.